x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/waterfall_flyout.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { EuiFlyout, EuiFlyoutHeader, EuiFlyoutBody, EuiTitle, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Table } from './waterfall_flyout_table';
+import { MiddleTruncatedText } from '../../waterfall';
+
+import { useWaterfallContext } from '../context/waterfall_chart';
+
+export const DETAILS = i18n.translate('xpack.uptime.synthetics.waterfall.flyout.details', {
```

Comment:
Is it preferred to put content in external content files, or within the component. Feel like I've seen it done both ways. Same with styled components.